users/views.py

```diff
@@ -11,3 +16,29 @@ def post(self, request):
         serializer.is_valid(raise_exception=True)
         serializer.save()
         return Response(serializer.data)
+
+
+class LoginView(APIView):
+    def post(self, request):
+        username = request.data["username"]
```

Comment:
instead of form-data, can we accept raw json instead? Let's add validation as well while we are at it